ui/src/Components/EditProfile.jsx

```diff
@@ -0,0 +1,266 @@
+import React from 'react';
+import { Button, Col, Form, FormGroup, Label, Input} from 'reactstrap';
+import NavBar from './NavBar';
+
+//Uncomment/Comment based on env
+// const URL = "http://localhost:3001/dev/editProfile";
+const URL = "https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/editProfile";
```

Comment:
we should change this var name to be like editProfileUrl or something along those lines to be more descriptive